### PR TITLE
Support more Zalo message types

### PR DESCRIPTION
**Pull request by khoa-connext**

### CODE REVIEW

File: zalo_sdk/BaseClient.py [hunk 1]
```diff
@@ -23,11 +23,12 @@ def access_token(self):
     def access_token(self, token):
         self._access_token = token
 
-    def send_request(self, method, url, body):
+    def send_request(self, method: str, url: str, body: dict, xtra_headers: str = {}):
```
Comment (quan-connext) [inline zalo_sdk/BaseClient.py:26]: typo? `xtra_headers`

File: zalo_sdk/oa/Client.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import requests
 import urllib.parse
 from zalo_sdk import BaseClient
-from zalo_sdk.oa import ZaloOAException
+import zalo_sdk.oa
 
 
 class Client(BaseClient):
```
Comment (quan-connext) [inline zalo_sdk/oa/Client.py:11]: Where is this endpoint on line 36? "https://oauth.zaloapp.com/v4/oa/permission"
I only found "https://oauth.zaloapp.com/v4/oa/access_token" in zalo docs 🤔 

https://developers.zalo.me/docs/api/official-account-api/xac-thuc-va-uy-quyen/cach-1-xac-thuc-voi-giao-thuc-oauth/lay-oa-access-token-tu-oa-refresh-token-post-4970

**Review by quan-connext — APPROVED** [review]

File: zalo_sdk/BaseClient.py (hunk 1 above)
Comment (khoa-connext) [reply zalo_sdk/BaseClient.py:26]: `xtra` is a shortcut for `extra` :P

https://www.dictionary.com/e/slang/xtra/

File: zalo_sdk/oa/Client.py (hunk 2 above)
Comment (khoa-connext) [reply zalo_sdk/oa/Client.py:11]: It's an undocumented feature, unfortunately. It's in this setting page https://developers.zalo.me/app/2230569949617529859/oa/settings

File: zalo_sdk/BaseClient.py (hunk 1 above)
Comment (quan-connext) [reply zalo_sdk/BaseClient.py:26]: whatttttt 🤯

**Pull request merged**